src/libs/terms-of-service-alerts.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash/fp'
+import { Fragment, useEffect } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import { Link } from 'src/components/common'
+import { Ajax } from 'src/libs/ajax'
+import * as Nav from 'src/libs/nav'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+
+
+const getNewTermsOfServiceNeedsAcceptingAlert = async termsOfServiceState => {
+  const shouldNotify = termsOfServiceState.isGracePeriodEnabled && !termsOfServiceState.userAcceptedTos
```

Comment:
Trying to wrap my head around this logic.  If I'm reading this correctly, it seems like the only time users will see this alert is if the grace period is enabled.  That very well may be what is intended and I'm misunderstanding things, but I was under the impression that we were going to do away with the grace period all together, which means when that happens, this alert will never have a chance to show up.  Is that right?  

Totally fine if the answer is, "Nope, this is correct", just asking you to double check this.

Reply:
You're reading this correctly! We've gotten feedback from other teams that we _very much need_ a grace period. So, I've made the code only display the alert if there's a grace period enabled and the user hasn't accepted the new terms. If there's no grace period enabled, they'll be forced to accept it or deny and log out.